docs/releases/creating-a-release.md

```diff
@@ -68,6 +68,21 @@ Use the [create-release-branch](https://github.com/Azure/communication-ui-librar
 
 ![Trigger release branch creation](../images/trigger-create-release-branch.png)
 
+### Step 1.3: Update the UI snapshot (beta-release only)
+
+Use the [create-release-branch](https://github.com/Azure/communication-ui-library/actions/workflows/create-release-branch.yml) github action to trigger the release branch creation workflow.
+
+After finishing creating release branch, you will need to trigger the UI snapshot update for the release branch:
+![image](https://github.com/Azure/communication-ui-library/assets/11863655/271d6973-5501-40f7-b506-d4f3c836a118)
+
+### Step 1.4: Notify the release thread about api.md update and UI snapshot update (beta-release only)
+
+After you finishing step 1.3, you can check recent lastest commits on release branch, there should be 
```

Comment:
This would be better as a PR instead of commits directly to the branch

Reply:
(side note: snapshots would be probably too much added time, but should we add a CI build and unit test for beta flavor? So our ci would do stable, beta and alpha for each PR?)

Reply:
This is a point I wanna bring up in the CCC walk through session meeting later, ideally those snapshots and functions are in the middle of beta(main)/stable (it's included in the diff of beta vs stable), so they are fully tested already. Adding tests to these features wouldn't be hard, but a little bit wasting resources and engineering time, but I am not against, just need to pick up the right one for the team

And the PR one is a good idea with a great balance in the middle! I will add this to my notes and probably update it later